content/docs/programs/requisitions.md

```diff
@@ -103,6 +103,11 @@ You will now be redirected to the Requisition detail page. The lines will be
 pre-populated with the items from the program; this means that you cannot add
 items manually to the Requisition.
 
+#### Extra Requisition Fields
+
+In manual requisitions it is possible to collect many additional columns of data for reporting and forcasting purposes. To enable these columns you need to configure store permissions to enable these extra columns as documenting in mSupply Desktop documentation [here](https://docs.msupply.org.nz/other_stuff:virtual_stores#preferences_tab). For quick reference this is the preference your logged in store requires:
+![Show extra fields on requisitions preference](/docs/distribution/images/show_extra_fields_on_requisitions.png)
```

Comment:
This image seems broken for me?

Reply:
Try now? IDK how that was working for me but it certainly was!